.github/actions/verify-metrics-snapshot/action.yaml

```diff
@@ -44,17 +44,35 @@ runs:
         path: ./.metrics/cached_${{ inputs.snapshot}}
         key: ${{ env.v2_tag }}_${{ inputs.artifact_key }}
 
+    - name: Check cache and exit action
+      if: steps.cache-metrics.outputs.cache-hit != 'true'
+      shell: bash
+      run: |
+        echo "::notice::No cached metrics found for comparison, skipping remaining steps"
+        exit 78 
```

Comment:
what is 78?

Reply:
rather than exiting here, you can simply skip the comparison step if `cache-hit != true`